### PR TITLE
Implement WASM Support with increasing okhttp versions

### LINKED ISSUE #867: Web Assembly (wasm) Target
Blocking 

- Result (WIP)
- Okio/OkHTTP
- Kotlinx serialization

### PR DESCRIPTION
**Pull request by iNoles**
Close #867

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,18 +24,19 @@ use the `any http method` [suspend](https://kotlinlang.org/docs/reference/corout
 
 ```kotlin
 runBlocking {
-    val string = Fuel.get("https://publicobject.com/helloworld.txt").body.string()
+    val string: String = Fuel.get("https://publicobject.com/helloworld.txt").body.string()
```
Comment (kittinunf) [inline README.md:27]: [q] Do we need to specify type string here?

Comment (iNoles) [reply README.md:27]: Something about IntelliJ idea was having trouble to understand the types of it.

Comment (kittinunf) [reply README.md:27]: Small concern but it should be okay^^

**Review by kittinunf — APPROVED** [review]
Let's 👍

**Comment by kittinunf** [comment]
Do we wanna merge this in for alpha03 as well?

**Comment by iNoles** [comment]
> Do we wanna merge this in for alpha03 as well?

No, I'd rather wait.

**Comment by kittinunf** [comment]
Ok. Let's create a tag for alpha03 then

**Comment by iNoles** [comment]
> Ok. Let's create a tag for alpha03 then

Go Ahead! 👍

**Pull request merged**